tb_plugin/torch_tb_profiler/profiler/run_generator.py

```diff
@@ -197,7 +199,7 @@ def _generate_op_pie(self, group_by_input_shape=False):
         return data
 
     def _generate_op_table(self, group_by_input_shape=False):
-        show_gpu = self.profile_data.is_gpu_used
+        show_gpu = self.profile_data.has_kernel or self.profile_data.has_memory
```

Comment:
why not has_runtime?

Reply:
Because operator view could only show kernel, memcpy and memset. Whether runtime exists or not, it doesn't care.